radicle-node/radicle-node.service

```diff
@@ -0,0 +1,26 @@
+# When running radicle-node on a server, it should be run as a separate user.
+# Copy this file into /etc/systemd/system and set the User/Group parameters
+# under [Service] appropriately.
+
+# For users wishing to run a configured version of the service, copy this
+# service file into the user's systemd directory, and edit appropriately.
+#
+#   mkdir -p $HOME/.local/share/systemd/user/
+#   cp radicle-node.service $HOME/.local/share/systemd/user/radicle-node.service
+#
+# You will have to set the RAD_PASSPHRASE for the node to start.
+#
+#   [Service]
+#   Environment="RAD_PASSPHRASE=[passphrase]"
+
+[Unit]
+After=syslog.target network.target
+Description=Radicle server daemon
+
+[Service]
+ExecStart=/usr/sbin/radicle-node
```

Comment:
```suggestion
ExecStart=%h/.radicle/bin/radicle-node
```
We won't be installing this in system folders, so this would make the most sense I think.

Reply:
How about "./local/bin"? It is recommended in [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html).  Though better places may be `/usr/local/sbin` or `/opt/radicle/bin/radicle`.

"User-specific executable files may be stored in $HOME/.local/bin. Distributions should ensure this directory shows up in the UNIX $PATH environment variable, at an appropriate place.

Since $HOME might be shared between systems of different achitectures, installing compiled binaries to $HOME/.local/bin could cause problems when used on systems of differing architectures. This is often not a problem, but the fact that $HOME becomes partially achitecture-specific if compiled binaries are placed in it should be kept in mind."

Reply:
We don't use XDG standards though, eg. we have everything else already under `~/.radicle`

Reply:
Are there other applications which do this?  I haven't seen this done or recommended before.  The other standards I'd recommend `/opt/radicle/bin/radicle-node` or `/usr/local/bin` which are common approaches.

Reply:
There are some yes, for example `cargo` puts binaries in ~/.cargo/bin, and `go` puts them in `$GOPATH`.

Reply:
<img width="270" alt="image" src="https://user-images.githubusercontent.com/40774/228843797-34e4cbb6-f804-4b8d-964a-b596ca66180f.png">

Reply:
Is that recommended though?  Radicle will be the only project I know of doing this officially.

Reply:
Cargo & rustup do it too as demonstrated. There's no better location that doesn't require administrator priviledges.

Reply:
Looks like we commented at the same time.

You're right, from their [page](https://www.rust-lang.org/tools/install):

```
In the Rust development environment, all tools are installed to the ~/.cargo/bin directory, and this is where you will find the Rust toolchain, including rustc, cargo, and rustup. 
```